internal/install/recipes/bundler.go

```diff
@@ -35,7 +34,7 @@ func (b *Bundler) CreateCoreBundle() *Bundle {
 func (b *Bundler) CreateAdditionalGuidedBundle() *Bundle {
 	var recipes []string
 
-	for _, d := range b.Detections {
+	for _, d := range b.AvailableRecipes {
```

Comment:
Random thought - instead of iterating over N recipes to remove the two that are part of the core bundle, why not remove the two core bundle recipes from AvailableRecipes since you can delete from a map by key?

Reply:
@mbruzina But we still need to loop through our map to create a slice?

Reply:
Ah, fair point 👍